src/components/Organisms/Donate/Form/Form.js

```diff
@@ -163,7 +164,7 @@ const Signup = ({
                 <MoneyBox
                   isSelected={amountDonate === value}
                   amount={value}
```

Comment:
Again, nothing major just a query. In the other instances we pass the `amountFormatter(value)` into the component, but here we handle this inside the component.

```js
const formattedAmount = amountFormatter(amount);
```

Was there any specific reason for this? Again just largely unimportant point.

Reply:
Ahhh nah, I think this was left over from me figuring out stuff, good catch. I'll move it out, and clear up the naming in this MoneyBuy/Moneybox thing, it just confused tf out of me

Reply:
Yeah, I found that confusing as well.